### PR TITLE
Move Prose and ProseWithHeading into ingredients

### PR DESCRIPTION
**Pull request by kenrick95**
As mentioned in https://github.com/mdn/stumptown-renderer/pull/172#issuecomment-538010513, I'm moving Prose and ProseWithHeading into ingredients folder

### CODE REVIEW

File: client/src/ingredients/prose.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+export function Prose({ section }) {
+  return <div dangerouslySetInnerHTML={{ __html: section.content }} />;
+}
+
+export function ProseWithHeading({ id, section }) {
+  if (!id) {
```
Comment (joedarc) [inline client/src/ingredients/prose.js]: By looking at document.js I don't believe this check is needed anymore?

```
if (!section.value.id || PROSE_NO_HEADING.includes(section.value.id)) {
  return <Prose key={section.value.id} section={section.value} />;
} else {
  return (
    <ProseWithHeading
      key={section.value.id}
      id={section.value.id}
      section={section.value}
    />
  );
}
```

Comment (joedarc) [reply client/src/ingredients/prose.js]: Also noticed that the key for `Prose` would be set to undefined if `section.value.id` does not exist.

Comment (kenrick95) [reply client/src/ingredients/prose.js]: I should've been reading what they are doing when moving them.

Alright, I've changed these two! Thanks for the feedback

Comment (joedarc) [reply client/src/ingredients/prose.js]: Thanks @kenrick95! I'd wait for @peterbe to make the call on this but..it looks like we use `ingredient-name${i}` for most ingredients but not all. Wondering if we should just make that the go-to for keys for ingredients across the board.

File: client/src/ingredients/prose.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+export function Prose({ section }) {
+  return <div dangerouslySetInnerHTML={{ __html: section.content }} />;
+}
+
+export function ProseWithHeading({ id, section }) {
+  return (
+    <>
+      <h2 id={id}>{section.title}</h2>
+      <div dangerouslySetInnerHTML={{ __html: section.content }} />
```
Comment (peterbe) [inline client/src/ingredients/prose.js]:
```suggestion
      <Prose section={section} />
```

Comment (peterbe) [reply client/src/ingredients/prose.js]: I know you just moved existing code. Probably written by me :)
But I noticed it now and it's a chance to get it right.

File: client/src/document.js [hunk 3]
```diff
@@ -2,6 +2,9 @@ import React from "react";
 import { Link } from "@reach/router";
 
 import { NoMatch } from "./routing";
+
+// Ingredients
```
Comment (peterbe) [inline client/src/document.js]: Isn't that obvious? :)

Comment (kenrick95) [reply client/src/document.js]: :joy: Sorry it's a habit to name a group when grouping imports together

**Pull request merged**